tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2172,6 +2172,20 @@ def test_email_wildcard_condition(self):
         assert len(response.data["data"]) == 1
         assert response.data["meta"]["message"] == "string"
 
+    def test_release_wildcard_condition(self):
+        release = self.create_release(version="test@1.2.3+123")
+
+        self.store_event(
+            data={"release": release.version, "timestamp": self.min_ago},
+            project_id=self.project.id,
+        ).event_id
```

Comment:
Is this `.event_id` leftover from some debugging?

Reply:
It is, thanks for catching that :+1: